tools/wpt/browser.py

```diff
@@ -1891,23 +1921,108 @@ class WebKitTestRunner(Browser):
     product = "wktr"
     requirements = None
 
-    def download(self, dest=None, channel=None, rename=None):
-        raise NotImplementedError
+    def _find_apple_port_builds(self, channel="main"):
+        if channel != "main":
+            raise ValueError(f"unable to get builds for branch {channel}")
 
-    def install(self, dest=None, channel=None):
-        raise NotImplementedError
+        system_version, _, _ = platform.mac_ver()
+        if system_version in SpecifierSet("==13.*"):
+            platform_key = "mac-ventura-x86_64%20arm64"
+        elif system_version in SpecifierSet("==12.*"):
+            platform_key = "mac-monterey-x86_64%20arm64"
+        else:
+            raise ValueError(
+                f"don't know what platform to use for macOS {system_version}"
+            )
 
-    def install_webdriver(self, dest=None, channel=None, browser_binary=None):
+        # This should match http://github.com/WebKit/WebKit/blob/main/Websites/webkit.org/wp-content/themes/webkit/build-archives.php
+        build_index = get(
+            f"https://q1tzqfy48e.execute-api.us-west-2.amazonaws.com/v3/latest/{platform_key}-release"
```

Comment:
This is a terrifyingly temporary looking URL which I'd prefer if we could look up somewhere, but ofc it's up to you.

Reply:
We've changed the `v3` twice in the past six years, I'm not super concerned about the stability here. Though yeah, it doesn't seem great…